### PR TITLE
feat: make the dashboard darker

### PR DESCRIPTION
**Pull request by kylecarbs**
Coder is the internal software development platform. It is not designed to be opinionated on colors, but it should look great.

Focusing on neutrality for our default dashboard theme is great for our ICP. Some organizations may lean towards colors more or less, and that shouldn't influence their decision when exploring Coder.

Before:
image

After:
image

### CODE REVIEW

**Comment by Parkreiner** [comment]
So I was a fan of the blue palette. It seemed to strike a good balance between being different, while looking restrained, since blue is generally treated as a more traditional color.

Muting the non-interactive colors further does have some benefits, though. It makes the elements that still have color pop out, because they're working more like true highlight colors. That can help give us more ways of emphasizing important elements

I'd be a little sad to see the blue go away, but I feel like it might give us more room overall for using color for functional purposes

**Comment by Parkreiner** [comment]
I also feel like we might not need to kill the color entirely, and just "condense" the color ranges. GitHub's palette reads as pretty neutral to me, but even their backgrounds are still blue-tinged

File: site/src/theme/colors.ts [hunk 1]
```diff
@@ -4,43 +4,43 @@ export const colors = {
...
-    9: "hsl(219, 28%, 52%)",
-    8: "hsl(218, 29%, 58%)",
-    7: "hsl(219, 30%, 64%)",
-    6: "hsl(219, 31%, 71%)",
-    5: "hsl(218, 32%, 77%)",
-    4: "hsl(223, 38%, 84%)",
-    3: "hsl(218, 44%, 92%)",
-    2: "hsl(220, 50%, 95%)",
-    1: "hsl(220, 55%, 98%)",
+    17: "hsl(0, 0%, 1%)",
```
Comment (Parkreiner) [inline site/src/theme/colors.ts:7]: Unrelated to the actual PR, but is there a reason why we have so many color stops? I think the most I've ever seen is 10–12 (usually mapping to the font weight system of 100-900, with maybe a 50 and a 950 thrown in)

Comment (kylecarbs) [reply site/src/theme/colors.ts:7]: I agree this is debt. We should reduce these in the future.

**Comment by kylecarbs** [comment]
@Parkreiner it's not that I wasn't a fan of the blue, I just think this darker palette is a better base for us to build on.

We can slightly tinge things blue as well, I'm fine with that, I just want something more base-dark for us to iterate on.

**Review by BrunoQuaresma — APPROVED** [review]
If storybook is good, I'm good 👍

**Pull request merged**